Azure/azure-cli
# [Compute] az vm availability-set list: Support showing VM list

**Description**  

https://github.com/Azure/azure-cli/issues/11821
`az vm availability-set list` does not show VM list in each availability set.
Now it can show it.

Before:
```
"virtualMachines": null
```

After:
```
"virtualMachines": [
      {
        "id": "/subscriptions/0b1f6471-1bf0-4dda-aec3-cb9272f09590/resourceGroups/FYTEST4/providers/Microsoft.Compute/virtualMachines/VM1",
        "resourceGroup": "FYTEST4"
      }
    ]
```


**Testing Guide**  

```
az vm availability-set create -g rg-n as
az vm create -g rg-n vm1 --image centos --availability-set as
az vm availability-set list
```
You should be able to see VM list in each availability set.

**History Notes**  


[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.  
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.
